packages/vue/src/components/organisms/SfHeader/SfHeader.stories.js

```diff
@@ -1,7 +1,18 @@
-import { SfHeader } from "@storefront-ui/vue";
+import {
+  SfHeader,
+  SfList,
+  SfBottomNavigation,
+  SfMegaMenu,
+  SfLink,
+  SfMenuItem,
+  SfBanner,
+  SfOverlay,
+} from "@storefront-ui/vue";
+
 export default {
   title: "Components/Organisms/Header",
   component: SfHeader,
+  parameters: { actions: { argTypesRegex: "^click.*" } },
```

Comment:
We don't need this here. In `preview.js` it's already enabled. you just need to use `argTypes` and pass the `action` field for the prop instead of control.

Reply:
I doesn't work on my local. I'll check it once again.